### ISSUE TITLE
Support for different language in title and description

### ISSUE THREAD

**Issue by pszafer** [iamcal/oembed#212]
Hi,
Does oEmbed suppport multiple language? In youtube from some time videos can have translated titles.
oEmbed always returning default language and as far as I see in specs it doesn't support titles and description. Is that true?

**Comment by iamcal** [comment]
that's true. the current spec does not have support for multiple languages

**Comment by pszafer** [comment]
Do you have plan to change it? can you do that?
